### ISSUE TITLE
MySQL Bug Cannot Insert null timestamps

### ISSUE THREAD

**Issue by IDragonfire** [FAForever/db#85]
Expected behaviour
https://bugs.mysql.com/bug.php?id=72628

```sql
mysql> create table t1(c1 timestamp not null default '2000-01-01 00:00:00');
Query OK, 0 rows affected (0.02 sec)

mysql> insert into t1 values();
Query OK, 1 row affected (0.00 sec)

mysql> insert into t1 values(null);
Query OK, 1 row affected (0.01 sec)

mysql> insert into t1 values(default);
Query OK, 1 row affected (0.00 sec)

mysql> select * from t1;
+---------------------+
| c1                  |
+---------------------+
| 2000-01-01 00:00:00 |
| 2014-08-01 06:11:50 |
| 2000-01-01 00:00:00 |
+---------------------+
3 rows in set (0.00 sec)
```

Behaviour with `5.7.16`
```sql

mysql> create table t1(c1 timestamp not null default '2000-01-01 00:00:00');
Query OK, 0 rows affected (0.22 sec)

mysql> insert into t1 values(null);
ERROR 1048 (23000): Column 'c1' cannot be null
```

**Comment by micheljung** [comment]
Please don't close issues as long as they haven't been merged into `develop`. Wafflebot will take care of it once a commit containing `Fix #85` is merged.

**Comment by IDragonfire** [comment]
# Hotfix (after 2017-05)
Recreate the docker container (then a newer mysql version is used)
